roles/hosted_engine_setup/tasks/bootstrap_local_vm/02_create_local_vm.yml

```diff
@@ -93,6 +100,10 @@
           {{ he_local_vm_dir }}/network-config
         environment: "{{ he_cmd_lang }}"
         changed_when: true
+      - name: Adjust ISO file permissions
+        file:
+          path: "{{ he_local_vm_dir }}/seed.iso"
+          mode: 0644
```

Comment:
same

Reply:
This task has been removed, but still keep it 0644 for the iso file (as part of `Fix local VM directory permission` task) otherwise the deployment fails with the same error.